shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/API/Validation.hs

```diff
@@ -52,7 +52,7 @@ chainChecks ::
   ) =>
   Globals ->
   PParams era ->
```

Comment:
Consensus needs access to the `PParams` when calling `chainChecks`. Before this PR, we could extract them from the `LedgerView`, but that's no longer possible with this PR. Consensus doesn't have anything else in scope where it could get the `PParams` from.

Potential solution: create a record with the fields needed by `chainChecks` (`_maxBHSize`, `_maxBBSize`, `_protocolVersion`), let `chainChecks` that record as an argument and store the record in the `LedgerView`.

(Without a fix, the integration in consensus is broken.)